packages/components/organisms/f-takeawaypay-activation/src/components/ActivationSuccessful.vue

```diff
@@ -46,19 +46,6 @@ export default {
 
 <style lang="scss" module>
 
-.c-takeawaypayActivation-icon {
-    transform: translate(10%);
-    width: 125px;
-}
-
-.c-takeawaypayActivation-title {
-    @include font-size(body-l);
-    font-weight: $font-weight-bold;
-    padding-top: spacing(x2);
```

Comment:
Not sure if you need to add a line to cover this for this component specifically?

Reply:
@ashleynolan Sorry, I didn't get it. Could you please rephrase? 🙂

Reply:
Thanks @ashleynolan. It was really needed for all components, so added it back.